README.md

```diff
@@ -105,6 +105,18 @@ By default, this will create a set of certificates in `/etc/grafana` that are ba
 
 _Note:_ Enabling SSL on Grafana will not allow for running on privileged ports such as `443`. To enable this capability you can use the suggestions documented in [this Grafana documentation](http://docs.grafana.org/installation/configuration/#http-port)
 
+### Collect metrics from postgres
+
+To allow telegraf to connect to postgres, apply the included profile class to the postgres host:
+
+```
+class {'puppet_metrics_dashboard::profile::postgres':
+  pg_password => 'changeme',
+}
+```
+
+Additionally, in the main `puppet_metrics_dashborad` class, you'll need to pass values for `postgres_host` and 'postgres_password` (must match value of `pg_password` above)
```

Comment:
If these parameters should be the same, can we set the default in the profile to be the init class?

Reply:
good idea!